eventhubs-streamanalytics-eventhubs/tools/eh-asa-perfmon/eh-asa-perfmon.csproj

```diff
@@ -1,14 +1,16 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp2.1</TargetFramework>
```

Comment:
Any reason that we're not using `netcoreapp3.1`?   That is the current LTS version and recommended version when developers [download the SDK](https://dotnet.microsoft.com/download/dotnet-core).   That would allow the `LangVersion` to be `latest` instead of taking a preview dependency.  I think that would be a better pattern to demonstrate if there's no constraint forcing us to use 2.1

Reply:
resolved.